src/project_profiles.ts

```diff
@@ -47,24 +47,21 @@ export interface IProjectProfile {
 }
 
 export function getProfilesFromDetection(projectPoolDir?: string): IProjectProfile[] {
```

Comment:
`getProfilesFromDetection` and `getDeletedProfilesFromDetection` can be effectively merged into a single function.